sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBServerVersion.cs

```diff
@@ -22,16 +22,10 @@ public CosmosDBServerVersion(string value)
             _value = value ?? throw new ArgumentNullException(nameof(value));
         }
 
-        private const string Three2Value = "3.2";
-        private const string Three6Value = "3.6";
-        private const string Four0Value = "4.0";
-
-        /// <summary> 3.2. </summary>
-        public static CosmosDBServerVersion Three2 { get; } = new CosmosDBServerVersion(Three2Value);
-        /// <summary> 3.6. </summary>
-        public static CosmosDBServerVersion Three6 { get; } = new CosmosDBServerVersion(Three6Value);
-        /// <summary> 4.0. </summary>
-        public static CosmosDBServerVersion Four0 { get; } = new CosmosDBServerVersion(Four0Value);
+        private const string V3_2Value = "3.2";
+        private const string V3_6Value = "3.6";
+        private const string V4_0Value = "4.0";
+        private const string V4_2Value = "4.2";
```

Comment:
Interesting, why we don't have `4.2` in the previous generated code?

Reply:
Arthur updated the swagger commit

Reply:
That's the problem that we not use `readme.md` under spec repo